tests/constants.py

```diff
@@ -0,0 +1,22 @@
+from utils import construct_path
+
+AGGREGATION_MODE = 0
+CAIRO_PATH = ["contracts/src"]
+DEFAULT_DECIMALS = 18
```

Comment:
This goes out the window given the new approach?

Reply:
I have a small comment on this.  What do we return if there's no decimals set for a currency?  Should it just be 0?

Reply:
https://github.com/42labs/Empiric/pull/161/files#diff-dfd5978dbc5000df40ee978202e1865a783ced4d8fe4fb72017939c83cdefa78R182